channels/tsmf/client/gstreamer/tsmf_gstreamer.c

```diff
@@ -43,10 +43,14 @@
 #include <inttypes.h>
 #endif
 
+// 1 second
+#define SEEK_TOLERANCE 10000000
```

Comment:
Oh, I think I have other comments that need to be multiline as well, I can do your suggestion to make the value more readable.